specifyweb/frontend/js_src/lib/components/Forms/SubView.tsx

```diff
@@ -204,7 +204,14 @@ export function SubView({
         <Button.BorderedGray
           aria-label={relationship.label}
           aria-pressed={isOpen}
-          className="w-fit"
+          className={`
+            w-fit 
+            ${
+              (collection?.models.length ?? 0) > 0
+                ? 'border-2 !border-brand-500'
```

Comment:
while these colors work in dark mode, they likely look bad in light mode. can you double-check?